documentation/swagger/api.yaml

```diff
@@ -290,6 +290,33 @@ paths:
                 type: "object"
       security:
         - basicAuth: []
+  /group:
+    post:
+      summary: "Create an group"
+      description: ""
+      consumes:
+        - "application/json"
+      produces:
+        - "application/json"
+      parameters:
+        - in: "body"
+          name: "body"
+          description: "Name of Organisation"
+          required: true
+          schema:
+            $ref: "#/definitions/Organisation"
```

Comment:
probably Group

Reply:
why is it for parameters in organisation post it says scheme,
I thought this was create as one of the parameter is organisation for group.